src/lib/dataAccess.js

```diff
@@ -1,26 +1,38 @@
 import axios from "axios";
 
 const baseURL = process.env.REACT_APP_ADMIN_BACKEND_URL;
+axios.defaults.withCredentials = true;
```

Comment:
Do you need this line? Each of the below requests also have `withCredentials: true`

Reply:
Doesn't look like it (just tested). Will remove